src/components/Search/Search.js

```diff
@@ -32,6 +32,7 @@ const Search = ({ determineSearchResults }) => {
     <form className='search' onSubmit={handleSubmit}>
       <input
         type='text'
+        value={searchInput}
```

Comment:
Not sure how I missed this but thank you for catching that!

Reply:
Yep, I'd fixed this on my branch too and talked through it here with Cole, so this should help with those state updates 🤘